pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -301,7 +301,9 @@ type StretchClusterZoneSpec struct {
 
 // MgrSpec represents options to configure a ceph mgr
 type MgrSpec struct {
-	Modules []Module `json:"modules,omitempty"`
+	Count                int      `json:"count,omitempty"`
+	AllowMultiplePerNode bool     `json:"allowMultiplePerNode,omitempty"`
```

Comment:
Can't we just use anti-affinity by default just like we do for MDS instead and let users decide?

Reply:
Anti-affinity is added by default. If this is `true`, it is `preferred`, but if this is false it is `required`, the same pattern as the mons. If it's just preferred, it allows to still run on a single node. Or what choice is needed for the users?